Fails to build

This is possibly due to the git dependencies. Did you consider commiting the Cargo.lock file to the repo? :)

Setting up the github workflow to [run periodically](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events) could also help, but might be unnecessary with the lock file in the repo.

Thanks!

```
error[E0277]: the trait bound `MkvDemuxer: format::demuxer::Demuxer` is not satisfied
   --> src/main.rs:138:34
    |
138 |         let mut c = Context::new(Box::new(MkvDemuxer::new()), Box::new(ar));
    |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `format::demuxer::Demuxer` is not implemented for `MkvDemuxer`
    |
    = note: required for the cast to the object type `dyn format::demuxer::Demuxer`

error[E0308]: mismatched types
   --> src/main.rs:186:41
    |
186 |                         dec.send_packet(&pkt).unwrap(); // TODO report error
    |                                         ^^^^ expected struct `av_data::packet::Packet`, found struct `data::packet::Packet`
    |
    = note: expected reference `&av_data::packet::Packet`
               found reference `&data::packet::Packet`
    = note: perhaps two different versions of crate `av_data` are being used?

error[E0308]: mismatched types
   --> src/main.rs:187:28
    |
187 |                         Ok(dec.receive_frame().ok())
    |                            ^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `data::frame::Frame`, found struct `av_data::frame::Frame`
    |
    = note: expected enum `Option<Arc<data::frame::Frame>>`
               found enum `Option<Arc<av_data::frame::Frame>>`
    = note: perhaps two different versions of crate `av_data` are being used?

error: aborting due to 3 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,9 +11,9 @@ keywords = ["player","rust-av","sdl2"]
 [dependencies]
 clap = "2.24.1"
 sdl2 = "0.34"
-av-data = "0.2.0"
+av-data = "0.3.0"
 av-codec = "0.2.0"
-av-format = "0.2.0"
+av-format = "0.3.0"
 libvpx = { version = "0.1.0", git = "https://github.com/rust-av/vpx-rs", features = ["codec-trait"] }
 libopus = { version = "0.1.0", git = "https://github.com/rust-av/opus-rs", features = ["codec-trait"] }
 matroska = { version = "0.1.0", git = "https://github.com/rust-av/matroska" }

